app/models/user_cue.rb

```diff
@@ -1,4 +1,7 @@
 class UserCue < ApplicationRecord
   belongs_to :user
```

Comment:
belongs to user through accounts instead?

Reply:
if we do this then if you delete an account you would lose all cues you have set for that account + this way it seems faster to query all cues from user and the other way around as well

Reply:
Cool!